controllers/beer.ctrl.js

```diff
@@ -23,60 +22,53 @@ module.exports.getBeer = async (req, res) => {
     const beer = await Beer.findByPk(beer_id);
 
     if (!beer) {
-      res.status(404).json({
-        error: "Bière introuvable."
-      });
+      res.status(404).json({ error: Errors.BEER_NOT_FOUND });
     }
+    
     res.status(200).json(beer);
   } catch (error) {
-    res.status(500).json({
-      error: "Une erreur est survenue lors de la récupération de la bière."
-    })
+    res.status(500).json({ error: Errors.BEER_GET });
   }
 };
 
 module.exports.deleteBeer = async (req, res) => {
-  const beer_id = req.params.beer_id;
+  const { beer_id } = req.params;
 
   try {
     const beer = await Beer.findByPk(beer_id);
 
     if (!beer) {
-      return res.status(404).json({ error: "La bière n'existe pas" });
+      return res.status(404).json({ error: Errors.BEER_NOT_FOUND });
     }
 
     await beer.destroy();
-
+    
     res
       .status(200)
-      .json({ message: `La bière avec l'ID ${beer_id} a été supprimée` });
+      .json({ message: `La bière avec l'ID ${beer_id} a bien été supprimée.` });
   } catch (error) {
-    res.status(500).json({ error: "La suppression de la bière a échoué" });
+    res.status(500).json({ error: Errors.BEER_DELETE });
   }
 };
 
 module.exports.createBeer = async (req, res) => {
-  const id_bar = req.params.id_bar;
+  const bar_id = req.params.bar_id;
```

Comment:
Gardons une cohérence et si possible, le mettre dans le try catch
```suggestion
  const { bar_id } = req.params;
```

Reply:
Oui, c'est pour ça que je l'ai renommé. On s'était décidé pour du <nom>_id. Du coup, partout où c'était nécessaire j'ai renommé.